tests/system_tests/test_artifacts/test_artifact_public_api.py

```diff
@@ -89,17 +89,20 @@ def test_artifact_files(user, api, sample_data, relay_server, inject_graphql_res
     assert paths[0].startswith("wandb_artifacts/")
 
     # Assert we don't break legacy local installs
-    server_info_response = inject_graphql_response(
-        # request
-        query_match_fn=lambda query, variables: query.startswith("query ServerInfo"),
-        # response
-        body="""{"data": {"serverInfo": {"cliVersionInfo": {"max_cli_version": "0.12.20"}}}}""",
+    gql = wandb_backend_spy.gql
+    responder = gql.once(
+        content="""{"data": {"serverInfo": {"cliVersionInfo": {"max_cli_version": "0.12.20"}}}}""",
+        status=200,
```

Comment:
`content` accepts a dict:

```suggestion
    responder = gql.once(
        content={"data": {"serverInfo": {"cliVersionInfo": {"max_cli_version": "0.12.20"}}}},
        status=200,
```